country_by_country/img_table_extraction/camelot_extractor.py

```diff
@@ -30,25 +31,30 @@
 class Camelot:
     def __init__(self, flavor: str) -> None:
         self.flavor = flavor
+        self.type = "camelot"
 
-    def __call__(self, pdf_filepath: str, assets: dict) -> None:
+    def __call__(self, pdf_filepath: str) -> dict:
```

Comment:
The asset concept seems fairly central.  It could be good to define a class or pydantic model for this.